Templates/Tests/Expected/AutoMockable.expected

```diff
@@ -33,28 +33,27 @@ import AppKit
 
 
 
-public class AccessLevelProtocolMock: AccessLevelProtocol {
+class AccessLevelProtocolMock: AccessLevelProtocol {
 
-    public init() {}
 
-    public var company: String?
```

Comment:
Hey @rokridi , I wonder why public modifier is removed. It might be a regression

Reply:
fixed